gradle.template.properties

```diff
@@ -4,6 +4,7 @@
 org.gradle.daemon=false
```

Comment:
Can remind me of the purpose of this line? Mine is always set to true to avoid the warning that builds will be slower.

Reply:
In CI, with `daemon = true` the log messages from the dev server (`./gradlew appengineStart`) will contaminate the console when running the tests (`./gradlew ciTests`).